index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    <div class="container">
```

Comment:
I would sugested the wrapping entire form tag in a `main` tag, for better semantics, and overall readabilty, and you should be good to go. 👌🏾